release/create_release_charts.sh

```diff
@@ -90,6 +91,18 @@ function copy_versions_files() {
     cp "${OPERATOR_BASE_DIR}/version/versions.yaml" "${OUTPUT_DIR}"
 }
 
+function update_version() {
+    local DOCKER_HUB="docker.io/istio"
+    local DOCKER_TAG=${OPERATOR_VERSION}
```

Comment:
I'm concerned putting this into the script will make it hard to understand what's going on when it's called. 
Also, I'm a bit confused about where docker.io and gcr.io should be used.